app/src/main/res/layout/quest_surface_detailed_answer_impossible.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:divider="@drawable/space_8dp"
+    android:showDividers="middle">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/quest_surface_detailed_answer_impossible_description"/>
+
+    <EditText
+        android:id="@+id/explanationInput"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="textMultiLine"
+        android:gravity="top|start"
+        android:minLines="3"
+        tools:text="Surface is a mosaic of paving stones, wood and metal."
+        android:maxLength="255"
+        android:scrollHorizontally="false"
+        android:autofillHints="Tap here to write."
```

Comment:
`autofillHints` does something else than you think it does. I think you meant `hint`. But `hint` should be localized, too. A reasonable hint would be to give an example what should be written here.
On the other hand, `quest_surface_detailed_answer_impossible_description` already serves this purpose, so I think this can be removed.

```suggestion
```